Gopkg.toml

```diff
@@ -42,8 +47,7 @@ ignored = ["github.com/fnproject/fn/cli",
 
 [[constraint]]
   name = "github.com/sirupsen/logrus"
-  revision = "89742aefa4b206dcf400792f3bd35b542998eb3b"
-
+  revision= "v1.0.6"
```

Comment:
since we're here https://github.com/sirupsen/logrus/blob/master/CHANGELOG.md v1.1.0 has some good looking bugs, like fixing a race for formatting